superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.tsx

```diff
@@ -95,6 +95,28 @@ const collapseStyles = (theme: SupersetTheme) => css`
   }
 `;
 
+const NO_DATABASES_MATCH_TITLE = t('No databases match your search');
+const NO_DATABASES_AVAILABLE_TITLE = t('There are no databases available');
+const MANAGE_YOUR_DATABASES_TEXT = t('Manage your databases');
+const HERE_TEXT = t('here');
+
+export const emptyStateComponent = (emptyResultsWithSearch: boolean) => (
```

Comment:
If this is going to be exported, I think that we should move it out of the SQlEditorLeftBar? Where are the other empty state components stored? THat's where it should be.

Reply:
Good thinking! Done in [`this commit`](https://github.com/apache/superset/pull/22262/commits/985f6a919628df1135aa322ff08510b18af6ca3d).